pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -610,6 +610,8 @@ func (cloud *CloudProvider) AttachDisk(ctx context.Context, volKey *meta.Key, re
 	op, err := cloud.service.Instances.AttachDisk(cloud.project, instanceZone, instanceName, attachedDiskV1).Context(ctx).Do()
 	if err != nil {
 		return fmt.Errorf("failed cloud service attach disk call: %v", err)
+	} else {
+		klog.V(5).Infof("Attaching disk %s operation id is ", volKey, op.Name)
```

Comment:
Also log op.Name at line 618? There may be lots of logging between these two line so it would be easier to debug a failed waitForZonalOp if we had the op name.

Reply:
added, thanks!